src/app/pipes/propertydisplay.ts

```diff
@@ -1,5 +1,6 @@
 import { PipeTransform, Pipe } from '@angular/core';
 import { DatePipe } from '@angular/common';
+import { isEmpty } from 'rxjs/operators';
```

Comment:
Could you please remove this unneeded import? VSCode probably automatically imported this when you tab-completed something.